stingray/tests/test_stats.py

```diff
@@ -33,6 +33,19 @@ def test_fold_detection_level():
     )
 
 
+def test_fold_detection_level():
```

Comment:
Duplicate name?

Reply:
Yup, should be `test_pdm_detection_level`